src/prefect/server/schemas/core.py

```diff
@@ -543,6 +556,12 @@ class Deployment(ORMBaseModel):
     is_schedule_active: bool = Field(
         default=True, description="Whether or not the deployment schedule is active."
     )
+    paused: bool = Field(
+        default=False, description="Whether or not the deployment is paused."
```

Comment:
I'm sorry if I missed this, but what is the reason for tracking if there are no active schedules, or what we are calling a "paused" deployment ?

Reply:
There's a long write up in the notion raft doc: https://www.notion.so/prefect/Enhanced-Scheduling-ca193685508c437eb84d05df957e8efa?pvs=4#ad67aa8695cd4198a2a6291a14ccd791

Reply:
ah interesting! i did not realize we have converged upon this point:

>  A paused deployment can’t be used to run an ad-hoc run or scheduled runs.

but that's neat and definitely a change from what currently happens when you press the pause button. do you know if we'll have some sort of warning about that?

Reply:
because if someone has a paused deployment currently and a bunch of ad hoc runs going, won't those be unable to run?

Reply:
Yeah, that bit is in flux. There's discussion within the scheduling design 'sub-raft', but being able to maintain the state of schedules when pausing/resuming a deployment itself is, IMO, enough to warrant the 'paused' field itself.

Reply:
maintaining the state of schedules through pause changes is definitely cool. I'm just not quite seeing here at least in this PR how it will prevent an ad hoc flow run.

Reply:
which makes sense then as you said it's still in flux.